pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -943,7 +956,7 @@ private ParsedBinaryTimestamp toParsedTimestampBin(TimeZone tz, byte[] bytes, bo
    * @return The parsed local date time object.
    * @throws PSQLException If binary format could not be parsed.
    */
-  public LocalDateTime toLocalDateTimeBin(byte[] bytes)  throws PSQLException {
+  public LocalDateTime toLocalDateTimeBin(TimeZone tz, byte[] bytes) throws PSQLException {
```

Comment:
Yes, but you are right that something is incorrect: the new `tz` parameter should be passed to `toParsedTimestampBin`.